project/build.properties

```diff
@@ -1 +1 @@
-sbt.version=1.3.13
+sbt.version=1.4.9
```

Comment:
Try sbt 1.5.2 if we it doesn't throw startup or compile errors

Reply:
throws additional errors